README.md

```diff
@@ -1,6 +1,6 @@
 # pixmo-docs
 
-This is the repository for the generation system of the [Pixmo-docs](https://huggingface.co/datasets/allenai/pixmo-docs) dataset, which support the generation of synthetic charts, tables, diagrams and more.
+This is the repository for the generation system of the [PixMo-Docs](https://huggingface.co/datasets/allenai/CoSyn-400K) and [PixMo-Docs Point](https://huggingface.co/datasets/allenai/CoSyn-point) datasets, which support the generation of synthetic charts, tables, diagrams and more. The older v0 datasets used to train [Molmo](https://arxiv.org/abs/2409.17146) can be [here](https://huggingface.co/datasets/allenai/pixmo-docs).
```

Comment:
The naming is confusing since the paper/HF use the CoSyn name. Maybe this could be more like:

This is the repository for the generation system of the [Pixmo-docs],  [CoSyn-400K], and [CoSyn-point] datasets. Pixmo-docs was used to train the Molmo model, and the CoSyn datasets are an expanded version that uses an improved pipeline and more types of documents.

Reply:
Yeah, agreed on the naming scheme, updated.